languages/common-lisp/exercises/concept/basic-lists/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+Leslie the Lisp Alien needs to do some shopping. It is very important to have a shopping list. One needs to add things to it, remove things from it, and even look if something is on the list for instance.
+
+Of course simple pen and paper will not do for a Lisp Alien. "List" is most of the word "Lisp" even!. There must be some functions written to help keep track of the shopping.
+
+Can you help Leslie keep track of the shopping list?
+
+## 1. Making a new list
+
+First thing is that Leslie needs to create a new empty list. A function called `empty-list` would be perfect for that.
+
+Oh no... Leslie actually has a few things in mind already so she needs a function that takes a three items (luckily Leslie never creates a list if she has less or more than three items) and creates a new shopping list with those things. Write a function called `list-of-things` which will take three items and makes a list of them.
+
```

Comment:
Could you add an example of the function call to the task? See [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd) and [this existing Common Lisp exercise](https://github.com/exercism/v3/blob/master/languages/common-lisp/exercises/concept/basics/.docs/instructions.md).